### ISSUE TITLE
`dnt` (Do Not Track) option for Vimeo

### ISSUE THREAD

**Issue by adrianbj** [blynx/TextformatterVideoEmbedOptions#5]
Hi @blynx - would you please add a `dnt` option for Vimeo?

More info: https://www.thomasvantuycom.com/privacy-friendly-video-embeds/

Thanks.

**Comment by netcarver** [comment]
@blynx  Hi Steffen, This would be a nice feature to have.

**Comment by blynx** [comment]
Hi!
indeed - ugh, and I just woke up from a overly long processwire sleep.

I created this PR: https://github.com/blynx/TextformatterVideoEmbedOptions/pull/6, but I just edited it quickly here in github. Could you have a look and see if it works for you? I try to get my hands back on an processwire instance to check that out soon I hope.

(I think this should just make the option available in the module settings - maybe there are more new options to adapt actually ...!?)

Also, looking at https://stackoverflow.com/questions/63173525/vimeo-embed-do-not-track and following the concers to https://stackoverflow.com/questions/44837450/recommended-method-to-prevent-any-content-inside-iframe-from-setting-cookies there could be implemented some fancier control.

**Comment by blynx** [comment]
Oh and btw, does anyone of you want to help maintain this module (and maybe another one ... !?) Since I'm not having my hands on processwire right now and probably wont have any time soon.
(Will get back into the forums later, too, to see how to hand over things)

**Comment by netcarver** [comment]
@blynx Hi Steffen, thanks for waking up for this :) Will try out your changes later this afternoon.

Do you have a roadmap or other changes in mind for this module?

**Comment by blynx** [comment]
Not really, I just thought there might be other new properties like dnt which could be added. So I/we could have just a look at the vimeo/youtube docs again and see whats new and could be added here.
I mean this module does have a quite narrow 'feature space'(!?) ... so I think the most changes would be some refinements every now and then.

**Comment by adrianbj** [comment]
@blynx - I found that it didn't work for me with the default `"true"` but if I changed it to `1` then it worked as expected. Do you know of any reason not to go with `1`?

**Comment by blynx** [comment]
ah mh, seemed ok from the vimeo docs. I just changed it.
